From c8e424344b41e42b42624885cec9f50382984622 Mon Sep 17 00:00:00 2001
From: John Ralls <jralls@ceridwen.us>
Date: Sat, 6 Dec 2025 15:45:23 -0800
Subject: [PATCH] Bug 799702 - build breaks with guile 3.0.11

Guile 3.0.11 has a new implementation of SRFI-64 that is more strict about
what test-group-with-cleanup accepts as parameters and it revealed a
couple of misplaced closing parens.
---
 gnucash/report/test/test-commodity-utils.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnucash/report/test/test-commodity-utils.scm b/gnucash/report/test/test-commodity-utils.scm
index 76e12ed2daa..33b197275c7 100644
--- a/gnucash/report/test/test-commodity-utils.scm
+++ b/gnucash/report/test/test-commodity-utils.scm
@@ -753,9 +753,9 @@
            (exchange-fn
             (gnc:make-gnc-monetary AAPL 1)
             USD
-            (gnc-dmy2time64-neutral 20 02 2012)))))
+            (gnc-dmy2time64-neutral 20 02 2012))))))
 
-      (teardown))))
+      (teardown)))
 
 (define (test-get-match-commodity-splits-sorted)
   (test-group-with-cleanup "test-get-match-commodity-splits-sorted"
@@ -818,6 +818,6 @@
                       "2012-02-20" "2013-08-09" "2014-07-11" "2014-08-08"
                       "2014-08-08" "2014-12-05" "2014-12-05" "2015-04-02"
                       "2015-10-23" "2015-10-23" "2016-03-11" "2016-03-11")
-                    (map split->date all-splits)))
+                    (map split->date all-splits))))
 
-      (teardown))))
+      (teardown)))
